src/Core/Import/Handler/ImportHandlerInterface.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Import\Handler;
+
+use PrestaShop\PrestaShop\Core\Import\Configuration\ImportConfigInterface;
+use PrestaShop\PrestaShop\Core\Import\Configuration\ImportRuntimeConfigInterface;
+use PrestaShop\PrestaShop\Core\Import\Exception\EmptyDataRowException;
+use PrestaShop\PrestaShop\Core\Import\File\DataRow\DataRowInterface;
+
+/**
+ * Interface ImportHandlerInterface describes an import handler.
+ */
+interface ImportHandlerInterface
+{
+    /**
+     * Executed before import process is started.
+     *
+     * @param ImportConfigInterface $importConfig
+     * @param ImportRuntimeConfigInterface $runtimeConfig
+     */
+    public function setUp(ImportConfigInterface $importConfig, ImportRuntimeConfigInterface $runtimeConfig);
```

Comment:
All imports need these methods at the moment, but only because it's needed for legacy logic to be executed. 

Anyway, I still think it's good to have those methods in the core interface because if you check this interface, it has an `importRow()` method, so that method is called once for every row that's being imported. `setUp()` and `tearDown()` are called once per whole import process, so it can be used to run hooks in the future, write information to logs, send emails or something else that's needed to be executed only once.

wdyt @matks?